.circleci/config.yml

```
@@ -21,10 +18,12 @@ jobs:
             - ~/.cache/pre-commit
   release:
     docker:
-      - image: *circleci_docker_primary
+      - image: goreleaser/goreleaser
```

should we tag the image version

goreleaser/goreleaser:v0.128
```suggestion
      - image: goreleaser/goreleaser:v0.128
```

---

probably!